app/controllers/post_controller.rb

```diff
@@ -1,18 +1,17 @@
 class PostController < ApplicationController
-  before_action :set_user
-  before_action :set_post, except: %i[index new create]
-  before_action :set_post, only: %i[show edit update destroy]
+  before_action :set_user, only: [:index]
 
   def index
     @posts = Post.all
```

Comment:
According to the requirements, you were expected to  replace `@posts = Post.all` with `@posts = @user.posts.includes(:comments)` before starting the integrations, consider doing so in order to satisfy the milestone requirement